convert2rhel/breadcrumbs.py

```diff
@@ -87,8 +89,10 @@ def finish_collection(self, success=False):
             self._set_target_os()
 
         self._set_ended()
+
         self._save_migration_results()
-        self._save_rhsm_facts()
+        if self._inform_telemetry != False and "CONVERT2RHEL_DISABLE_TELEMETRY" not in os.environ:
```

Comment:
I think this could be just 

```suggestion
        if self._inform_telemetry and "CONVERT2RHEL_DISABLE_TELEMETRY" not in os.environ:
```

If we `´self._inform_telemetry` is set to True, then this part of the condition will be evaluated as True.

Reply:
I mean, the idea here is that when `self._inform_telemetry = True` and the envvar is not present, right?

Reply:
Yes, you are right! Thanks!